photon-client/src/main/scala/com/linkedin/photon/ml/data/avro/AvroDataReader.scala

```diff
@@ -279,7 +274,7 @@ object AvroDataReader {
           val nameAndTerm = AvroUtils.readNameAndTermFromGenericRecord(record)
           val featureKey = Utils.getFeatureKey(nameAndTerm.name, nameAndTerm.term)
 
-          featureKey -> Utils.getDoubleAvro(record, AvroFieldNames.VALUE)
+          featureKey -> Utils.getDoubleAvro(record, FieldNames.VALUE)
```

Comment:
That's yet another `FieldNames`, local to class `AvroDataReader` :-) I'll try to cleanup.